relay-server/src/service.rs

```diff
@@ -143,6 +145,12 @@ impl ServiceState {
         let outcome_aggregator = OutcomeAggregator::new(&config, outcome_producer.recipient());
         registry.set(outcome_aggregator.start());
 
+        if let Ok(aws_runtime_api) = env::var("AWS_LAMBDA_RUNTIME_API") {
```

Comment:
_nit_: Move this into a (documented) function in the `relay-aws-extension` crate.

Reply:
For as long as we're just experimenting (if not for longer), I would strongly prefer a config flag for this in addition to the envvar. There is no telling what would happen if an existing, working customer relay is updated to include this code.

Reply:
added a `--aws-api-url=<URL>` cli arg